### DISCUSSION TITLE
Prime NG p-filter unsual behaviour when dynamically adding new columns for some columns the filter dosen't show options in the dropdown

### DISCUSSION THREAD

**Discussion by haseeb151994** [primefaces/community#1595] [PrimeNG]
I have been working dynamically adding columns to ptable and i have about 59 columns which i want to show on the first load time only 8 or 9 columns are shown according to user configuration but after that when i add columns dynamically most of the columns shows filters and their options in the dropdown but two columns usually don't work and they have string type filters on them and their filters option dropdown shows empty and when i press the filter dropdown an error is generated that cannot read properties of undefined(reading matchmode)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?
